xbmc/platform/darwin/tvos/XBMCApplication.mm

```diff
@@ -48,7 +53,15 @@ - (void)applicationWillTerminate:(UIApplication*)application
 
 - (void)applicationDidBecomeActive:(UIApplication*)application
 {
+  // Always occurs on the first start of Kodi
+  // and also after an applicationWillResignActive() or an applicationDidEnterBackground()
```

Comment:
`applicationDidEnterBackground` is always preceded by `applicationWillResignActive`, so the sentence is a bit wrong